docs/language-server/protocol-language-server.md

```diff
@@ -324,7 +324,7 @@ interface ExpressionUpdate {
   /**
    * The updated pointer to the method call.
    */
-  methodPointer?: SuggestionId;
+  methodPointer?: MethodPointer;
```

Comment:
This is the (major) API change: Instead of using `SuggestionId` to identify a method, we are using `MethodPointer`.